app/forms/respondent_form.rb

```diff
@@ -57,7 +57,8 @@ class RespondentForm < Form
 
   validates :no_acas_number_reason,
     inclusion: { in: NO_ACAS_REASON, allow_blank: true },
-    presence: { if: -> { no_acas_number? } }
+    presence: { if: -> { no_acas_number? } },
+            ccd_acas_exemption_reason: true
```

Comment:
have you tried this ?  If you specify an acas number I think it will still try to validate the ccd acas exemption reason